libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1147,6 +1147,9 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 		else if (member == "sig")
 			m_context << u256(0) << Instruction::CALLDATALOAD
 				<< (u256(0xffffffff) << (256 - 32)) << Instruction::AND;
+		else if (member == "blockhash")
```

Comment:
Is `member != "blockhash"` better and not having an empty block? I don't care which verison, this seem to be showing the intent better for someone looking at it first.

Reply:
It is more uniform this way, I would say.